include/swift/AST/Attr.def

```diff
@@ -494,7 +494,7 @@ SIMPLE_DECL_ATTR(_implementationOnly, ImplementationOnly,
   ABIStableToAdd | ABIStableToRemove | APIStableToAdd | APIStableToRemove,
   84)
 DECL_ATTR(_custom, Custom,
-  OnAnyDecl | UserInaccessible |
+  OnAnyDecl | RejectByParser |
```

Comment:
Were you able to find out *why* it was this way originally?

Reply:
No, but I _guess_ this was just a confusion between `RejectByParser` and `UserInaccessible`

Reply:
Yes, I wasn't thinking about the distinction carefully back when I did this. Thank you!